shared/search/index.js

```diff
@@ -31,7 +32,7 @@ class Search extends Component<void, Props, void> {
   }
 }
 
-const connector: TypedConnector<TypedState, TypedDispatch<SearchActions>, OwnProps, Props> = new TypedConnector()
+const connector: TypedConnector<TypedState, TypedDispatch<SearchActions | FSOpen>, OwnProps, Props> = new TypedConnector()
```

Comment:
i would explore the openInWindows changes if i didn't have to load windows/virtualbox... :)